README.md

```diff
@@ -1,7 +1,57 @@
-Looking to embed Tabzilla in your website? Check out the documentation at http://bedrock.readthedocs.org/en/latest/tabzilla.html.
+# Tabzilla
 
----
+This is the home of Tabzilla, the universal tab you can add to your website. Below
+follows simple instructions on how to get, and use Tabzilla.
 
-Tabzilla is no longer maintained in this separate repo. It is now integrated into the Bedrock repo:
+## Bower Package
 
-http://github.com/mozilla/bedrock/
+Do you use [Bower](http://bower.io/) to manage your front-end dependencies? Well, then you will be
+happy to hear that you can install Tabzilla using Bower.
+
+```Shell
+bower install mozilla-tabzilla
+```
+
+That's it! Now you have all the bits you need. Next step, add it to your site.
+
+## Manual Download
+
+Don't use [Bower](http://bower.io/)? No problem, you can [download the latest release right here from the releases page](https://github.com/mozilla/tabzilla/releases).
+
+Once downloaded, extract the package and add the `css` and `media` folders to your
+project.
+
+## Using Tabzilla
+
+First step is to add the `tabzilla.css` file to you main `HTML` file:
+
+```HTML
+<link rel="stylesheet" type="text/css" href="css/tabzilla.css" media="screen" />
+```
+
+Next, you need to add the following required `HTML` to your document.
+
+```HTML
+<div id="tabzilla">
+    <a href="https://mozilla.org">Mozilla</a>
+</div>
+```
+
+## Changing The Color of the Wordmark
+
+By default the color of the wordmark on Tabzilla is transparent, as below:
+
+![Tabzilla with transparent wordmark](http://cl.ly/image/0N1K213s1z40)
```

Comment:
Ok, I have updated this and added the images to the repo and I am now embedding them instead of the external images. With that said, it will not load until this is merged :-/ because the URL points to raw.github.com/mozilla/tabzilla....